### ISSUE TITLE
Publications screen headings

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#312]
When using voiceover on Publications screen:

Latest, Earlier today, dates should be identified as headings.
